packages/analytics/amplify_analytics_pinpoint/example/backend/bin/backend.ts

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// A copy of the License is located at
+//
+//  http://aws.amazon.com/apache2.0
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import { BackendStack } from '../lib/backend-stack';
+
+const app = new cdk.App();
+new BackendStack(app, 'BackendStack');
```

Comment:
request: Could you give this a more descriptive name so we can find it in cloudformation?